packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_screen.dart

```diff
@@ -53,114 +55,197 @@ class _DeepLinkPageState extends State<DeepLinkPage>
-                      controller: controller,
-                      showSpitScreen: showSpitScreen,
+                  ),
+            Expanded(
+              child: Row(
+                children: [
+                  Expanded(
+                    child: _AllDeepLinkDataTable(controller: controller),
+                  ),
+                  if (showSplitScreen)
+                    Expanded(
+                      child: ValueListenableBuilder<LinkData?>(
+                        valueListenable: controller.selectedLink,
+                        builder: (context, selectedLink, _) => TabBarView(
+                          children: [
+                            _ValidationDetailScreen(
+                              linkData: selectedLink!,
+                              controller: controller,
+                              tableView: TableViewType.domainView,
+                            ),
+                            _ValidationDetailScreen(
+                              linkData: selectedLink,
+                              controller: controller,
+                              tableView: TableViewType.pathView,
+                            ),
+                            _ValidationDetailScreen(
+                              linkData: selectedLink,
+                              controller: controller,
+                              tableView: TableViewType.singleUrlView,
+                            ),
+                          ],
+                        ),
+                      ),
                     ),
-                  ],
-                ),
+                ],
               ),
             ),
-          ),
-        ],
+          ],
+        ),
       ),
     );
   }
 }
 
-class _DataTableWithValidationDetails extends StatelessWidget {
-  const _DataTableWithValidationDetails({
-    required this.linkDatas,
-    required this.tableView,
+class _AllDeepLinkDataTable extends StatelessWidget {
+  const _AllDeepLinkDataTable({
     required this.controller,
-    required this.showSpitScreen,
   });
-  final List<LinkData> linkDatas;
-  final TableViewType tableView;
   final DeepLinksController controller;
-  final bool showSpitScreen;
 
   @override
   Widget build(BuildContext context) {
-    return Row(
-      children: [
-        Expanded(
-          child: _DataTable(
-            tableView: tableView,
-            linkDatas: linkDatas,
-            controller: controller,
+    final theme = Theme.of(context);
+    final colorScheme = theme.colorScheme;
+    final textTheme = theme.textTheme;
+    return Column(
+      children: <Widget>[
+        Container(
+          decoration: BoxDecoration(
+            border: Border.fromBorderSide(
+              defaultBorderSide(Theme.of(context)),
+            ),
+            color: colorScheme.surface,
+          ),
+          padding: const EdgeInsets.fromLTRB(
+            defaultSpacing,
+            denseSpacing,
+            denseSpacing,
+            denseSpacing,
```

Comment:
seems odd that we are using a non-uniform border only on the left. If the extra 8 pixels is to pad the entire `_AllDeepLinkDataTable` widget to the left, then the padding should be placed at that level of the tree.